### DISCUSSION TITLE
A new mutation for plants

### DISCUSSION THREAD

**Discussion by 12347654** [cataclysmbn/Cataclysm-BN#3826] [General]
### Describe the solution you'd like

How about a mutation that would allow players to grow food on themselves? For example we spend 500 calories and get a fruit with 100 that doesn't spoil for weeks or months.

**Comment by Proxiehunter** [comment]
Sacrificing calories when they're plentiful so you can bank them for later sounds useful and in theme. Definitely need to make sure it doesn't trivialize hunting/farming (speaking of farming maybe a later mutation on the tree could allow getting a seed from this fruit to _grow_ something. A friendly triffid maybe?) but I think a five to one ration might be _too_ low.
